core/src/main/kotlin/com/malinskiy/marathon/Marathon.kt

```diff
@@ -79,6 +80,10 @@ class Marathon(
         val tests = applyTestFilters(parsedTests)
         val shard = prepareTestShard(tests, analytics)
 
+        log.info("\n\n\n **** Marathon File Params **** \n")
```

Comment:
This is a no-no: configuration might contain sensitive credentials, and we can't print them to the stdout

Reply:
If you really want to do this then we need to implement some kind of masking for sensitive fields